### PR TITLE
PMM-5020 add tablestat

### PR DESCRIPTION
**Pull request by lunaticusgreen**
[PMM-5020](https://jira.percona.com/browse/PMM-5020)

### CODE REVIEW

File: pmm-app/src/pmm-add-instance/components/AddRemoteInstance/FormParts/AdditionalOptions/AdditionalOptions.tsx [hunk 1]
```diff
@@ -24,27 +29,68 @@ export const AdditionalOptionsFormPart: FC<AdditionalOptionsFormPartProps> = ({
...
+      component={RadioButtonGroupAdapter}
+    />
+  </>
 );
-export const getAdditionalOptions = (type, remoteInstanceCredentials) => {
+
+const MySQLOptions = ({ form }) => {
+  const [selectedValue, setSelectedValue] = useState<string>(TablestatOptions.disabled);
+
+  const getTablestatValues = (type) => {
```
Comment (tiagomotasantos) [inline pmm-app/src/pmm-add-instance/components/AddRemoteInstance/FormParts/AdditionalOptions/AdditionalOptions.tsx:53]: Function can be moved to outside of the component.

File: pmm-app/src/pmm-add-instance/components/AddRemoteInstance/FormParts/FormParts.types.ts [hunk 2]
```diff
@@ -8,6 +8,7 @@ export interface AdditionalOptionsFormPartProps {
   instanceType?: string;
   loading: boolean;
   remoteInstanceCredentials: RemoteInstanceCredentials;
+  form: any;
```
Comment (tiagomotasantos) [inline pmm-app/src/pmm-add-instance/components/AddRemoteInstance/FormParts/FormParts.types.ts:11]: You can use FormApi from final-form here.

**Review by tiagomotasantos — CHANGES_REQUESTED** [review]

File: pmm-app/src/pmm-add-instance/components/AddRemoteInstance/FormParts/AdditionalOptions/AdditionalOptions.tsx (hunk 1 above)
Comment (lunaticusgreen) [reply pmm-app/src/pmm-add-instance/components/AddRemoteInstance/FormParts/AdditionalOptions/AdditionalOptions.tsx]: done

File: pmm-app/src/pmm-add-instance/components/AddRemoteInstance/FormParts/FormParts.types.ts (hunk 2 above)
Comment (lunaticusgreen) [reply pmm-app/src/pmm-add-instance/components/AddRemoteInstance/FormParts/FormParts.types.ts]: done

File: pmm-app/src/pmm-add-instance/components/AddRemoteInstance/FormParts/FormParts.test.tsx [hunk 3]
```diff
@@ -75,7 +76,7 @@ describe('AdditionalOptionsFormPart ::', () => {
             instanceType={type}
             remoteInstanceCredentials={remoteInstanceCredentials}
             loading={false}
-            form={form}
+            form={(form as unknown) as FormApi}
```
Comment (tiagomotasantos) [inline pmm-app/src/pmm-add-instance/components/AddRemoteInstance/FormParts/FormParts.test.tsx:79]: Can use the form from render props

**Review by tiagomotasantos — APPROVED** [review]

**Pull request merged**